### ISSUE TITLE
creating a child card is not 'sticking'

### ISSUE THREAD

**Issue by adappt-io** [h-be/acorn-release#28]
I've set up Acorn on two Ubuntu VMs with two accounts, and the second account was able to join the first account's project.   The first card (via G + click) works and is visible in both VMs.

Attaching a second card doesn't seem to stay on the canvas, from either account.  As per instructions:
1. Click the first card --> blue border added to the red border
2. Go to canvas and press G + click --> second card appears
3. Add title to the second card and press either Enter or the canvas --> second card disappears.

Seems consistent (haven't succeeded in creating the second card yet).

**Comment by adappt-io** [comment]
No I lied.   I exited Acorn and then started it again, and all the child cards I entered are now there.  So the issue is no refresh after doing step 3 above.

**Comment by Connoropolous** [comment]
Hey @adappt-io 

so after your second update, can you summarize the issue for me again, if there still is one?

**Comment by Connoropolous** [comment]
when something like what you described happens, it is usually a sign that Holochain has went somewhat unresponsive, since it is waiting for a response from Holochain to render the card. 

I haven't done as much testing on the linux system as the Mac one, so there could also be something's up with that particular setup. 

This was with the latest release?
